src/index.html

```diff
@@ -4,8 +4,21 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <div id="todo_section"></div>
+    <!-- <form id="form">
+        <div class="header">
+            <span>Today's To Do</span>
+            <span class="material-symbols-outlined">
+                cached
+                </span>
+        </div><hr>
+        <input class="input" type="text" placeholder="Add to your list...">
+        <ul id="todos_list"></ul>
+        <small>Clear all completed</small>
+    </form> -->
```

Comment:
- [ ] [OPTIONAL] As best practice, removing unused codes completely is a great deal. It prevents your work from looking messy or difficult to parse. Kindly remove this if you have no need of it anymore :+1: